CMakeLists.txt

```diff
@@ -26,6 +27,8 @@ IF (MSVC)
     SET(COMPILER "v120") # Visual Studio 2013
   ELSEIF (${MSVC_VERSION} EQUAL 1900)
     SET(COMPILER "v140") # Visual Studio 2015
+   ELSEIF (${MSVC_VERSION} EQUAL 1910)
+     SET(COMPILER "v141") # Visual Studio 2017    
```

Comment:
Well, according to this blog from Microsoft we both may be correct:
https://blogs.msdn.microsoft.com/vcblog/2016/02/24/stuck-on-an-older-toolset-version-move-to-visual-studio-2015-without-upgrading-your-toolset/

The versions number schema for Visual Studio is highly confusing:

| Visual Studio Product Name |  Product Version | Compiler Version |Toolset Version |
| ------------------- |----:| ----:|----:|
| Visual Studio 2008 | 9.0 | 1500 | V90 |
| Visual Studio 2010 | 10.0 | 1600 | V100 |
| Visual Studio 2012 | 11.0 | 1700 | V110 |
| Visual Studio 2013 | 12.0 | 1800 | V120 |
| Visual Studio 2015 | 14.0 | 1900 | V140 |
| Visual Studio 2017 | 15.0 | 1910 | V141 |

But I guess what I am actually referring to is the toolset version. So your version is probably the most correct, although I should probably rename the variable `COMPILER` to `TOOLSET`.